### ISSUE TITLE
Gef build fails with Java 11

### ISSUE THREAD

**Issue by vogella** [eclipse-gef/gef#37]
GEF build currently fails with Java 11.

To test, modify ~/git/gef/org.eclipse.gef.releng/pom.xml

Switch to Tycho 1.3.0 snapshot in linie 22
````
<tycho.version>1.3.0-SNAPSHOT</tycho.version>
````

Add tycho snapshot to `pluginRepositories`in line line 636

````
	<pluginRepositories>
	 	<pluginRepository>
      		<id>tycho-snapshots</id>
      		<url>https://repo.eclipse.org/content/repositories/tycho-snapshots/</url>
     	</pluginRepository>
		<pluginRepository>
			<id>CBI</id>
			<url>https://repo.eclipse.org/content/repositories/cbi-releases/</url>
		</pluginRepository>
	</pluginRepositories>
````

Now run `mvn clean verify -PNEON.target` on the command line -> unable to find Fx classes as they are not available anymore in Java 11.

**Comment by miklossy** [comment]
See also the corresponding [Eclipse GEF Forum post](https://www.eclipse.org/forums/index.php/t/1102788/).

**Comment by laeubi** [comment]
@vogella Zulu offers JDK Version with integrated FX: https://www.azul.com/downloads/zulu-community/?architecture=x86-64-bit&package=jdk-fx
Just make sure to add the required jar to the java installation in eclipse for development as described here: https://stackoverflow.com/questions/34284003/error-the-import-javafx-embed-swt-cannot-be-resolved-on-eclipse-plug-in-expor/49322093#49322093
